samples/tab-add-in-combined/nodejs/add-in/webpack.config.js

```diff
@@ -5,7 +5,7 @@ const CopyWebpackPlugin = require("copy-webpack-plugin");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 
 const urlDev = "https://localhost:3000/";
-const urlProd = "https://www.blueyonderairlines.com/"; // CHANGE THIS TO YOUR PRODUCTION DEPLOYMENT LOCATION
+const urlProd = "https://tabdiscounts.z13.web.core.windows.net/"; // CHANGE THIS TO YOUR PRODUCTION DEPLOYMENT LOCATION
```

Comment:
Did you mention this in our Readme file?

Reply:
We are not using urlDev and urlProduction

Reply:
Fixed